forge-gui/res/cardsfolder/upcoming/broodrage_mycoid.txt

```diff
@@ -0,0 +1,9 @@
+Name:Broodrage Mycoid
+ManaCost:3 B
+Types:Creature Fungus
+PT:4/3
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckDefinedPlayer$ You.descended | Execute$ TrigToken | TriggerDescription$ At the beginning of your end step, if you descended this turn, create a 1/1 black Fungus creature token with "This creature can't block." (You descended if a permanent card was put into your graveyard from anywhere.)
```

Comment:
wouldn't it be enough to do `ValidPlayer$ You.decended` ?

Reply:
Hmm, some triggers with `CheckDefinedPlayer$ You.isMonarch` might be refactored too

Reply:
I think the difference are Intervening If -Clause 

While you can lose the Monarch in response, you can't lose "descended this turn"